packages/frontend/components/scaffold-eth/Balance.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import { useBalance } from "wagmi";
+
+type BalanceProps = {
+  address: string;
+  price: number;
+  minimized?: boolean;
```

Comment:
I'd remove this option for now.

We just want to show the balance for a given address in a nice format + the eth/usd conversion (I know we are leaving this for now).

If showing the address it's something we need in the future, we'll add it!